src/js/components/project-view/empty-project.tsx

```diff
@@ -15,9 +16,72 @@ const EmptyProject = ({ project }: Props) => {
   return (
     <section className="container">
       <SectionTitle><span>Get started</span></SectionTitle>
-      <div className="row center-xs">
+      <div className="row start-xs">
         <div className="col-xs-12 col-md-8 col-lg-6">
-          <SetupInstructions project={project} styles={styles} hideLabels={true} />
+              <div className={styles.instructions}>
+            <div className={styles.section}>
+              <div className={styles.text}>
+                The URL for this project's Git repository is
+                <div className={styles.url}>{project.repoUrl}</div>
+              </div>
+              <div className={styles.heading}>
+                <Icon className={styles['heading-icon']} name="file-o" />
+                I'm starting a new project
+              </div>
+              <div className={styles.text}>
+                Connect your project folder with
+              </div>
+              <div className={styles.code}>
+                <pre onClick={selectText}>
+                  git init<br />
+                  git add .<br />
+                  git commit -m “First commit”<br />
+                  git remote add origin {project.repoUrl}<br />
+                  git push -u origin master
+                </pre>
+              </div>
+
+              <div className={styles.heading}>
+                <Icon className={styles['heading-icon']} name="git-square" />
+                My project is already in a Git repository
+              </div>
+              <div className={styles.text}>
+                Add Minard as a new remote with
+              </div>
+              <div className={styles.code}>
+                <pre onClick={selectText}>
+                  git remote set-url ––add origin {project.repoUrl}<br />
```

Comment:
That's exactly the use case I'm going for, so maybe we should look at the texts again